test/soak/serve_hostnames/serve_hostnames.go

```diff
@@ -120,27 +141,36 @@ func main() {
 		for j := 0; j < *perNode; j++ {
 			podName := fmt.Sprintf("serve-hostname-%d-%d", i, j)
 			podNames = append(podNames, podName)
-			glog.Infof("Creating pod %s/%s on node %s", ns, podName, node.Name)
-			_, err := c.Pods(ns).Create(&api.Pod{
-				ObjectMeta: api.ObjectMeta{
-					Name: podName,
-					Labels: map[string]string{
-						"name": "serve-hostname",
+			// Make several attempts
```

Comment:
I would prefer to have it done sequentially -- it makes it easier for me to track what is going on in the log and there is no huge performance win since in general the test spends most of its time in the loop that issues requests.